CaPA tutorial updated

cabayer opened the pull request:
I took the time to update the CaPA tutorial. The main changes that have been incorporated for the tutorial to work properly include:

-  New Datamart file structure and naming conventions
- grib2 format changed so that we no longer select based on `stepType`, but instead use `parameterNumber` (this was the biggest headache to figure out)

This updated tutorial now works properly and can be shared with PT partners :)

cabayer committed:
CaPA tutorial updated

sergioredondo24 commented:
I confirm that the tutorial works properly once all libraries have been installed. On that note, my suggestion would be to list all libraries that need to be installed in the very first block. Also, the figure that gets plotted in [this block](https://github.com/NHS-Dev/nhs-tutorials/blob/update_capa_tutorial/python-examples/datamart-capa/datamart-capa-example.ipynb?short_path=62388d7#L3639) has _unknown[unknown]_ as _y_label_, its title is incomplete/hard to read, and has no comment or instruction.

cabayer committed:
creation of environment file and updated tutorial

cabayer commented:
Hi @sergioredondo24, thanks for taking the time to review! I made some changes based on your comments:
1. Created an `environment.yml` file so that the user can easily see what libraries are needed. I cleaned up the portion of text in the tutorial that was describing the needed libraries.
2. I fixed the figure you mentioned that had ugly axes. Hopefully you think it's a little prettier now :)
3. Corrected a typo where a plot said it was 6-hour accumulation when it was actually 24-hour accumulation.

@sergioredondo24 if you are able to give this one more look over before I merge it into main, I would appreciate that!

cabayer committed:
adding conda-forge as a channel to properly install cfgrib

sergioredondo24 committed:
Update datamart-capa-example.ipynb

Everything that I reviewed is working smoothly.
I added a recommendation to use a conda environment.
Also added a grid to the time series plot, and for it to get get the coordinates for the title from the dataset.
